src/Microsoft.ML.Core/Properties/AssemblyInfo.cs

```diff
@@ -99,4 +99,6 @@
 [assembly: InternalsVisibleTo(assemblyName: "TLC" + InternalPublicKey.Value)]
 [assembly: InternalsVisibleTo(assemblyName: "DotNetBridge" + InternalPublicKey.Value)]
 
+[assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.AutoML" + PublicKey.Value)]
```

Comment:
I would probably put this to line 50.
To have separation of Public and InternalPublic keys.
Because it's convenient to some extent.